config/application.rb

```diff
@@ -21,7 +21,7 @@
 module BookASecureMoveApi
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 7.1
+    config.load_defaults 8.0
```

Comment:
The Rails upgrade guide does not recommend changing this value on the first release, as I understand it. We're supposed to try it out with the values loaded from `config/initializers/new_framework_defaults_8_0.rb`, and when we know we don't need to roll back we can change `config.load_defaults`.